java/change-notes/2021-04-06-ssrf-query.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* The server-side request forgery (SSRF) query has been promoted from experimental to the main query pack. Its results will now appear by default. Thanks to original experimental query author @porcupineyhairs.
```

Comment:
```suggestion
* The query "Server Side Request Forgery (SSRF)" (`java/ssrf`) has been promoted from experimental to the main query pack. Its results will now appear by default. Thanks to original experimental query author @porcupineyhairs.
```
If it's "server-side" rather than "server side" then the query name should be updated in the query meta-data.

Reply:
Server-side is better grammar (it's a compound noun), but sources on the internet and in our codebase are inconsistent. I've switched to the hyphenated form throughout here.